code/datums/components/riding/riding.dm

```diff
@@ -127,7 +127,7 @@
 /datum/component/riding/proc/vehicle_turned(datum/source, _old_dir, new_dir)
 	SIGNAL_HANDLER
 
-	vehicle_moved(source, new_dir)
+	vehicle_moved(source, null, new_dir)
```

Comment:
because that would lead to dir delay as well? The vehicle now has a dir of new dir, let's turn the rider to new_dir as well

or are you asking about the null, in that case new_moved is asking for old_loc, which doesn't make sense here since loc didn't change

Reply:
nevermind I misread the argument, I thought it was olddir, not oldloc.